src/main/java/trivia/game/PlayerManagement.java

```diff
@@ -0,0 +1,43 @@
+package trivia.game;
+
+import trivia.models.Player;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayerManagement {
+
+    private Player active;
+    private long id = 1;
+    private List<Player> players = new ArrayList<>();
+
+    public void addOne(Player player){
+        player.setId(id);
+        players.add(player);
+        id++;
+    }
+
+    public Player getCurrentPlayer() {
+        checkId();
+        players.stream()
+                .forEach(player->{
+                    if(player.getId() == id) active = player;
```

Comment:
it's strange how this methods uses field member active: it changes the reference & returns it.